src/Elcodi/Plugin/TwitterBundle/Resources/views/Tweet/share.html.twig

```diff
@@ -0,0 +1,9 @@
+<a
+    href="https://twitter.com/share"
+    class="twitter-share-button"
+    data-url="{{ url }}"
+    data-text="{{ text }}"
```

Comment:
In case the content of this `text` variable is not safely escaped, consider using [the escape Twig filter](http://twig.sensiolabs.org/doc/filters/escape.html) as follows:

```twig
data-text="{{ text|escape('html_attr') }}"
```